wp-content/plugins/hurumap/src/Theme.js

```diff
@@ -39,6 +39,14 @@ export default createTheme({
         }
       }
     },
+    line: {
+      colorScale: ['#29a87c', '#a0d9b3', '#004494', '#4abc70'],
```

Comment:
Will these colours also be available in [Takwimu.AFRICA](/TakwimuAfrica/Takwimu.AFRICA)

Reply:
@kilemensi , yes I will add them there. These are from takwimu style guide